egov-persister/audit-service-persister.yml

```
@@ -33,4 +33,4 @@ serviceMaps:
 
             - jsonPath: $.AuditLogs.*.integrityHash
 
-            - jsonPath: $.AuditLogs.*.auditcorrelationid
\ No newline at end of file
+            - jsonPath: $.AuditLogs.*.auditCorrelationId
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)